wayland-server/src/staging.rs

```diff
@@ -0,0 +1,100 @@
+//! Staging protocols from wayland-protocols
```

Comment:
I assume this file being added was accidental?

Reply:
Yes...don't know how that got here..